articles/azure-large-instances/what-is-azure-large-instances.md

```diff
@@ -0,0 +1,194 @@
+
+## ALI benefits
+
+ALI is intended for critical workloads that require certification to run your enterprise applications. 
+Azure Large Instances implementations are dedicated only to you, and you'll have full access (root access) to the operating system (OS).
+You manage OS and application installation according to your needs.
+For security, the instances are provisioned within your Azure Virtual Network (VNet) with no internet connectivity.
+Only services running on your virtual machines (VMs), and other Azure services in same Tier 2 network, can communicate with your implementation of ALI.
+
+ALI offers these benefits:
+
+* Non-hypervised ALI, single tenant ownership
+* Low latency between Azure hosted application VMs to ALI implementations (0.35 ms)
+* All Flash SSD and NVMe
+  * Up to 1 PB/tenant
+  * IOPS up to 1.2 million/tenant
+  * 40/100-GB network bandwidth
+  * Accessible via FC
+* Redundant power, power supplies, NICs, TORs, ports, WANs, storage, and management
+* Hot spares for replacement on a failure (without the need for reconfiguring)
+* Customer-coordinated maintenance windows
+* Application-aware snapshots, archive, mirroring, and cloning
+
+## SKU availability in Azure regions
+
+ALI offers multiple SKUs certified for specialized workloads.
+Use the workload-specific SKUs to meet your needs:
+
+* Large instances – two-socket to four-socket systems
+* Very Large instances – 4-socket to 20-socket systems
+
+ALI for specialized workloads is available in the following Azure regions:
+
+* West Europe
+* North Europe
+* Germany West Central zones support
+* East US zones support
+* East US 2 zones support
+* West US zones support
+* West US 2 zones support
+* South Central US
+* South Central US
+
+> [!Note]
+>Zones support refers to availability zones in which a region where ALI can be deployed across zones for high resiliency and availability. This capability enables support for multi-site active-active scaling.
+
+## Managing ALI in Azure
+
+Depending on your needs, the application topologies of Azure Large Instances can be complex.
+You may deploy multiple instances in one or more locations. The instances can have shared or dedicated storage, and specialized LAN and WAN connections
+Therefore, for ALI, Azure offers a consultation with a CSA/GBB in the field who can work with you.
+
+By the time your ALI implementation has been provisioned, the OS, networks, storage volumes, placements in zones and regions, and WAN connections between locations have already been configured.
+You're set to register your OS licenses (BYOL), configure the OS, and install the application layer.
+
+You'll see all the Azure Large instance resources, and their state and attributes, in the Azure portal.
+You can also operate the instances and open service requests and support tickets from there.
+
+## ALI stamp 
+
+The Azure Large instance stamp itself combines the following components:
+
+* **Computing**
```

Comment:
```suggestion
* **Computing:**
```

Reply:
Formatted by design - it's meant to look like a header, but you can't have duplicate headers